library/tkinter.po

```diff
@@ -1407,6 +1808,14 @@ msgid ""
 "work fine; for other files, use raw reads or ``os.read(file.fileno(), "
 "maxbytecount)``."
 msgstr ""
+"Dans la mesure où vous ne savez pas combien d'octets sont disponibles en "
+"lecture, vous ne voudrez probablement pas utiliser les méthodes :class:`~io."
```

Comment:
```suggestion
"lecture, il ne faut pas utiliser les méthodes :class:`~io."
```
De manière générale, le français est plus directif que l'anglais. Ici le « may not want » serait mieux traduit par « il ne faut pas » ou « il ne vaut mieux pas », cf. https://github.com/python/python-docs-fr/issues/831